.pre-commit-config.yaml

```diff
@@ -1,28 +1,28 @@
 repos:
-  - repo: https://github.com/keep-network/pre-commit-golang.git
-    rev: 4cc73f21101f9da1208719b02bbbe0a4c491497e
-    hooks:
-      - id: go-imports
-      - id: go-vet
-      - id: go-lint
-  - repo: https://github.com/keep-network/pre-commit-hooks.git
-    rev: 63e729f
-    hooks:
-      - id: check-added-large-files
-  - repo: local
-    hooks:
-      - id: solium
-        name: Solidity linter
-        language: node
-        entry: solium
-        files: '\.sol$'
-        args:
-          - --dir=contracts/solidity/contracts/
-          - --config=contracts/solidity/.soliumrc.json
-        additional_dependencies:
-          - ethlint@1.2.3
-  - repo: https://github.com/prettier/prettier
-    rev: "master" # Use the sha or tag you want to point at
-    hooks:
-      - id: prettier
-        files: '\.js$'
\ No newline at end of file
+ - repo: https://github.com/keep-network/pre-commit-golang.git
+   rev: 4cc73f21101f9da1208719b02bbbe0a4c491497e
+   hooks:
+    - id: go-imports
+    - id: go-vet
+    - id: go-lint
+ - repo: https://github.com/keep-network/pre-commit-hooks.git
+   rev: 63e729f
+   hooks:
+    - id: check-added-large-files
+ - repo: local
+   hooks:
+    - id: solium
+      name: Solidity linter
+      language: node
+      entry: solium
+      files: '\.sol$'
+      args:
+      - --dir=solidity/contracts/
+      - --config=solidity/.soliumrc.json
+      additional_dependencies:
+      - ethlint@1.2.3
+ - repo: https://github.com/prettier/prettier
+   rev: "master" # Use the sha or tag you want to point at
+   hooks:
+    - id: prettier
+      files: '\.js$'
```

Comment:
Lol what happened here.

Reply:
Ah. No, we don't want prettier here, we want eslint… Here's how we do this in keep-ecdsa: https://github.com/keep-network/keep-ecdsa/blob/master/.pre-commit-config.yaml#L14-L19 .

Our eslint config also handles prettier formatting.